tests/biospecimen/test_biospecimen_resources.py

```diff
@@ -288,17 +291,24 @@ def test_filters(self):
         assert 8 == Diagnosis.query.count()
         assert 8 == Biospecimen.query.count()
         assert 5 == BiospecimenDiagnosis.query.count()
+        assert 6 == GenomicFile.query.count()
+        assert 8 == BiospecimenGenomicFile.query.count()
 
         # Create query - Participant p0, Diagnosis d2 has 2 biospecimens
         d = Diagnosis.query.filter_by(external_id='study0-p0-d2').first()
         s = Study.query.filter_by(external_id='s0').first()
         bds = BiospecimenDiagnosis.query.filter_by(
             diagnosis_id=d.kf_id).count()
+        gf = GenomicFile.query.filter_by(external_id='study0-b0-gf0').first()
+        bsgf = BiospecimenGenomicFile.query.filter_by(
+            genomic_file_id=gf.kf_id).count()
         assert bds == 2
+        assert bsgf == 2
 
         # Send get request
         filter_params = {'diagnosis_id': d.kf_id,
-                         'study_id': s.kf_id}
+                         'study_id': s.kf_id,
+                         'genomic_file_id': gf.kf_id}
```

Comment:
It may be worth testing these independently as it difficult to tell if each is working if the same results are returned for each in isolation.

Reply:
Yea prob a good idea for each filter to be tested independently and all together